frontend/src/index.css

```
@@ -10,11 +10,12 @@ html, body, #root {
     margin: 0;
 }
 
-body {
+#body-root {
     font-family: 'Century Gothic', CenturyGothic, AppleGothic, sans-serif;
```

Biome (2.1.2)

[error] Duplicate font names are redundant and unnecessary: CenturyGothic

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)